runtime/src/startup/mod.rs

```diff
@@ -29,10 +32,9 @@ macro_rules! set_main {
     {$name:ident} => {
         #[no_mangle]
         fn libtock_unsafe_main() -> ! {
-            use libtock_runtime::TockSyscalls;
-            let res = $name();
+            use $crate::TockSyscalls;
```

Comment:
I don't think you need `use $crate::TockSyscalls;` here anymore.

Reply:
Corrected.